packages/formation-react/src/components/Telephone/Telephone.mdx

```diff
@@ -126,18 +126,22 @@ import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/f
 |:----|:-------|:------------|
 | `222_VETS` | `8772228387` | VA Help Line |
 | `4AID_VET` | `8774243838` | National Call Center for Homeless Veterans |
+| `711` | `711` | Telecommunications Relay Service |
 | `911` | `911` | Emergency! |
 | `CAREGIVER` | `8552603274` | VA National Caregiver Support Line |
 | `CRISIS_LINE` | `8002738255` | Veterans Crisis hotline |
 | `CRISIS_TTY` | `8007994889` | Veterans Crisis hotline TTY |
-| `DS_LOGON` | `8005389552` | Defense Manpower Data Center |
+| `DMDC_DEERS` | `8663632883` | Defense Manpower Data Center (DMDC) + Defense Enrollment Eligibility Reporting System (DEERS) Support Office |
 | `DS_LOGON_TTY` | `8663632883` | Defense Manpower Data Center TTY |
+| `DS_LOGON` | `8005389552` | Defense Manpower Data Center |
+| `FEDERAL_RELAY_SERVICE` | `8008778339` | Federal Relay Service |
 | `GI_BILL` | `8884424551` | Education Call Center (1-888-GI-BILL-1) |
 | `GO_DIRECT` | `8003331795` | Go Direct/Direct Express (Treasury) |
 | `HELP_DESK` | `8555747286` | VA Help desk |
 | `HELP_TTY` | `8008778339` | VA Help Desk TTY |
 | `MY_HEALTHEVET` | `8773270022` | My HealtheVet help desk |
 | `NCA` | `8005351117` | National Cemetery Scheduling Office |
+| `SUICIDE_PREVENTION_LIFELINE` | `8007994889` | Suicide Prevention Line |
```

Comment:
Should we preemptively add `988`?

<img src="https://user-images.githubusercontent.com/136959/88087800-17c71780-cb4f-11ea-8ade-ed7193ed05cb.jpg" width="300" />

Reply:
Hmm, you cool if we wait until we actually get a use-case for it on vets-website? Just want to make sure we don't add unused constants